src/main/java/com/example/Asset/Management/model/AssetsModel.java

```diff
@@ -16,4 +18,6 @@ public class AssetsModel {
     private String assetName;
     private String assetModel;
     private Long assetQuantity;
+    private Date lastupdatedate;
```

Comment:
change datatype for lastupdatedate into LocalDateTime and also use java convension to declare variable